Modules/_elementtree.c

```diff
@@ -1023,24 +1023,15 @@ element_setstate_from_Python(ElementObject *self, PyObject *state)
 {
     static char *kwlist[] = {PICKLED_TAG, PICKLED_ATTRIB, PICKLED_TEXT,
                              PICKLED_TAIL, PICKLED_CHILDREN, 0};
-    PyObject *args;
     PyObject *tag, *attrib, *text, *tail, *children;
-    PyObject *retval;
 
     tag = attrib = text = tail = children = NULL;
-    args = PyTuple_New(0);
-    if (!args)
-        return NULL;
-
-    if (PyArg_ParseTupleAndKeywords(args, state, "|$OOOOO", kwlist, &tag,
-                                    &attrib, &text, &tail, &children))
-        retval = element_setstate_from_attributes(self, tag, attrib, text,
-                                                  tail, children);
-    else
-        retval = NULL;
 
-    Py_DECREF(args);
-    return retval;
+    if (!PyArg_ParseTupleAndKeywords(_PyTuple_Empty, state, "|$OOOOO", kwlist,
```

Comment:
This is the main purpose of this patch. If use the `PyTuple_New(0)` calls you need to save the result of `PyTuple_New(0)`, save the result of the function that uses an empty tuple, decref the saved result of `PyTuple_New(0)`. Two additional variables (or one variable and duplicated `Py_DECREF`), several additional lines. `_PyTuple_Empty` is added for convenience, it is convenient as `Py_None` or `Py_True`.